src/js/controller/model.js

```diff
@@ -229,6 +232,10 @@ define([
             // Mute the video if autostarting on mobile. Otherwise, honor the model's mute value
             _provider.mute(this.autoStartOnMobile() || _this.get('mute'));
 
+            // Set the playback rate to be the value that the provider supports and will play at
+            _provider.setPlaybackRate(_this.get('defaultPlaybackRate'));
```

Comment:
Wouldn't this accomplish the same thing as these 2 lines?
`this.setPlaybackRate(this.get('defaultPlaybackRate'));`